x-pack/plugins/ingest_manager/common/services/datasource_to_agent_datasource.ts

```diff
@@ -54,8 +54,9 @@ export const storedDatasourceToAgentDatasource = (
           streams: input.streams.map(stream => {
             const fullStream = {
               ...stream,
-              ...Object.entries(stream.config || {}).reduce(configReducer, {}),
+              ...stream.pkg_stream,
```

Comment:
why are we not including `stream.config`?

Reply:
the variables are used to populate the template, but I do not thing we should include them